packages/calcite-components/src/components/notice/notice.scss

```diff
@@ -3,13 +3,24 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-notice-width: The width of the component.
+ * @prop --calcite-notice-background-color: Specifies the background color of the component.
+ * @prop --calcite-notice-border-color: Specifies the border color of the component.
+ * @prop --calcite-notice-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-notice-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-notice-icon-color: Specifies the color of the component's icon property.
+ * @prop --calcite-notice-close-icon-color: Specifies the color of the component's `closable` icon color.
+ * @prop --calcite-notice-close-background-color: Specifies the color of the component's `closable` background color.
+ * @prop --calcite-notice-close-background-color-hover: Specifies the hover color of the component's `closable` background color.
```

Comment:
Remove state specific tokens. We can just update the single token based on the state of :host

Reply:
For these - it's for the internally rendered close button. Users won't be able to do `calcite-notice:hover { --calcite-notice-close-background-color: red }`. We need these because the close button also isn't transparently overlaying backgrounds (which, have their own issues for theming but less immediately need overrides).

Reply:
Same feedback as https://github.com/Esri/calcite-design-system/pull/8607#discussion_r1488278003

cc @jcfranco for confirmation.

Reply:
@alisonailea We discussed host-level states and agreed to not add tokens for these, but we missed this scenario (styling states of internal components/elements). We'd need to allow these to properly support theming, right?

Reply:
Ah yes that's true. For internal component state management we'll need to have state tokens. Thanks!

Reply:
> Ah yes that's true. For internal component state management we'll need to have state tokens. Thanks!

Wouldn't it be better to be consistent across all components rather than have some be able to define hover states via tokens and others where you need to use an existing component along with some host selector?